teams/teams-ps/teams/New-CsTeamsVoiceApplicationsPolicy.md

```diff
@@ -0,0 +1,216 @@
+---
+external help file: MicrosoftTeams-help.xml
+Module Name: MicrosoftTeams
+online version: https://docs.microsoft.com/en-us/powershell/module/skype/new-csteamsvoiceapplicationspolicy
+schema: 2.0.0
+---
+
+# New-CsTeamsVoiceApplicationsPolicy
+
+## SYNOPSIS
+Creates a new Teams voice applications policy. TeamsVoiceApplications policy governs what permissions the supervisors/users have over auto attendants and call queues.
+
+## SYNTAX
+
+```
+New-CsTeamsVoiceApplicationsPolicy [-Identity] <String> [-AllowAutoAttendantAfterHoursGreetingChange <Boolean>]
+ [-AllowAutoAttendantBusinessHoursGreetingChange <Boolean>]
+ [-AllowAutoAttendantHolidayGreetingChange <Boolean>] [-AllowCallQueueMusicOnHoldChange <Boolean>]
+ [-AllowCallQueueOverflowSharedVoicemailGreetingChange <Boolean>]
+ [-AllowCallQueueTimeoutSharedVoicemailGreetingChange <Boolean>]
+ [-AllowCallQueueWelcomeGreetingChange <Boolean>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+TeamsVoiceApplicationsPolicy is used for Supervisor Delegated Administration which allows tenant admins to permit certain users to make changes to auto attendant and call queue configurations. 
+
+
+## EXAMPLES
+
+### EXAMPLE 1
+```
+New-CsTeamsVoiceApplicationsPolicy -Identity SDA-Allow-Moh -AllowCallQueueMusicOnHoldChange $true
+```
+The command shown in Example 1 creates a new per-user Teams voice applications policy with the Identity SDA-Allow-Moh. This policy allows delegated administrators to change the music on hold information.
+
+### EXAMPLE 2
+```
+New-CsTeamsVoiceApplicationsPolicy -Identity SDA-Allow-AA-After-Hour -AllowAutoAttendantAfterHoursGreetingChange $true 
+```
+The command shown in Example 2 creates a new per-user Teams voice applications policy with the Identity SDA-Allow-AA-After-Hour. This policy allows delegated administrators to change after-hours greetings for auto attendants.
+
+
+## PARAMETERS
+
+### -Identity
+Unique identifier assigned to the policy when it was created.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: 2
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AllowAutoAttendantAfterHoursGreetingChange
+This property allows supervisor/user to change auto-attendant's after hours greeting. 
```

Comment:
usually we use the word **Parameter** when we are referring to the cmdlet (Opposite to when we are talking about cmdlet's outputted **properties**)